helm/opensource-config.yaml

```diff
@@ -86,6 +86,34 @@ dragonchain:
         requests:
           storage: 2Gi
 
+  # -- dragonchain.verificationNotifications --
+  # This Level 1 Dragonchain configuration will send received verifications via HTTP unique URL's configured here.
+  # Verifications will be sent with HTTP POST, and contain three relevant headers documented below.
+  # -- Headers --
+  # Authentication: Bearer ${signature}
+  # DragonchainId: PublicIdOfThisL1Dragonchain
+  # Content-Type: application/json
+  # -- Body --
+  # The post body will be JSON of the raw verification object received by this dragonchain. See Dragonchain documentation for details.
+  # -- Signature --
+  # The bearer token will be the signature of the utf8 encoded bytes of the post body signed with the private key, and verifiable with
+  # the public key found in the DragonchainId header. NOTE: Do not trust this header alone to verify signature. Make sure it's a public id
+  # that you trust and expect.
+  verificationNotification: null
```

Comment:
where's the counterpart for this to actually insert this into the env vars of the chain correctly? I don't see any edited helm file

Reply:
Notes: you also will need a default in the `values.yaml` of the chart itself for strict helm linting after you edit the helm chart.

Reply:
yup. Lost in a rebase. I have them coming up